qutebrowser/browser/commands.py

```diff
@@ -447,6 +500,7 @@ def tab_clone(self, bg=False, window=False):
         newtab.data.keep_icon = True
         newtab.history.deserialize(history)
         newtab.zoom.set_factor(curtab.zoom.factor())
+        new_tabbed_browser.set_tab_pinned(idx, pinned)
```

Comment:
I think it'd be cleaner to simply use `curtab.data.pinned` here instead of defining a variable above which you only use here.